src/main/java/frc/robot/subsystems/Intake2020.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.subsystems;
+/*
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
+*/
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.drive.TalonController;
+
+public class Intake2020 extends SubsystemBase {
+
+    TalonController arm = new TalonController(Constants.INTAKE2020_ARM_MOTOR_ID);
+    TalonController roller = new TalonController(Constants.INTAKE2020_ROLLER_MOTOR_ID);
+    
+    Boolean up = true;
+
+    public Intake2020() {
+      super();
+    }
+
+    //stage 1
+
+      /**
+       * Lowers the arm so that it can pick up balls.
+      */
+      public void lowerGrabber() {
+
+      arm.set(Constants.INTAKE2020_ARM_DOWN_SPEED);
+      up = false;
+
+    }
+
+    /**
+    * Raises the arm so that it's in the robot permeter for game start or when playing defense.
+    */
+    public void raiseGrabber() {
+
+      this.disableGrabber();
+      arm.set(Constants.INTAKE2020_ARM_UP_SPEED);
+      up = true;
+
+    }
+
+    //stage 2
+    /**
+    * set roller motor to regular speed so that it can pick up balls and place them into indexing
+    */
+    public void enableGrabber() {
```

Comment:
maybe you should use a better word then enable, also maybe call this the roller to stay consistent with your code

Reply:
I suggest names like `grabberIn` and `grabberOut`. Make it super clear so you can't misinterpret what it is doing.